baselines/diabetic_retinopathy_detection/experiments/config/imagenet21k_vit_base16_finetune.py

```diff
@@ -31,41 +31,50 @@ def get_config():
   """Config for training a patch-transformer on JFT."""
   config = ml_collections.ConfigDict()
 
-  # Fine-tuning dataset
-  config.data_dir = 'gs://ub-data/retinopathy'
-  # config.in_domain_dataset = 'ub_diabetic_retinopathy_detection'
-  # config.ood_dataset = 'aptos'
-  # config.train_split = 'train'
-  # config.val_split = 'validation'
+  # * Data load / output flags. *
 
-  # TODO(nband): fix issue with sigmoid loss.
-  config.num_classes = 2
+  # The directory where the model weights and training/evaluation summaries
+  #   are stored.
+  config.output_dir = (
+    '/tmp/diabetic_retinopathy_detection/vit-16-i21k/deterministic')
 
-  # BATCH_SIZE = 64  # pylint: disable=invalid-name
-  # config.batch_size = BATCH_SIZE
+  # Dataset for model pretraining. Specifies the config to use.
+  config.pretrain_dataset = 'imagenet21k'
 
-  # config.total_steps = 10_000
+  # Fine-tuning dataset.
+  config.data_dir = 'gs://ub-data/retinopathy'
 
-  # Input resolution of each retina image. (Default: 512)
-  config.pp_input_res = 512  # pylint: disable=invalid-name
-  pp_common = f'|onehot({config.num_classes})'
-  config.pp_train = (
-      f'diabetic_retinopathy_preprocess({config.pp_input_res})' + pp_common)
-  config.pp_eval = (
-      f'diabetic_retinopathy_preprocess({config.pp_input_res})' + pp_common)
+  # REQUIRED: distribution shift.
+  # 'aptos': loads APTOS (India) OOD validation and test datasets.
+  #   Kaggle/EyePACS in-domain datasets are unchanged.
+  # 'severity': uses DiabeticRetinopathySeverityShift dataset, a subdivision
+  #   of the Kaggle/EyePACS dataset to hold out clinical severity labels as OOD.
+  config.distribution_shift = 'aptos'
 
-  config.shuffle_buffer_size = 15_000  # Per host, so small-ish is ok.
-
-  config.log_training_steps = 100
-  config.log_eval_steps = 1000
-  # NOTE: eval is very fast O(seconds) so it's fine to run it often.
-  config.checkpoint_steps = 1000
-  config.checkpoint_timeout = 1
+  # If provided, resume training and/or conduct evaluation using this
+  #   checkpoint. Will only be used if the output_dir does not already
+  #   contain a checkpointed model. See `checkpoint_utils.py`.
+  config.resume_checkpoint_path = None
 
   config.prefetch_to_device = 2
   config.trial = 0
 
-  # Model section
+  # * Logging and hyperparameter tuning. *
```

Comment:
nit: let's keep with the style of `# Model section`. That is, no period at the end, and no `* text *` formatting.

same elsewhere in the config